tests/test_build_material.py

```
@@ -0,0 +1,223 @@
+def test_build_material_languages_empty_by_default() -> None:
+    material = build_material("2026-04-28-登壇メモ", "本文．", "文字起こし．")
+    assert material.languages == []
+
+
+def test_build_material_author_null_by_default() -> None:
+    material = build_material("2026-04-28-登壇メモ", "本文．", "文字起こし．")
+    assert material.author is None
+
+
+def test_build_material_author_from_memo_frontmatter() -> None:
+    memo = '---\nauthor: "Shota Nishihara"\n---\n\n本文．'
```

Ruff (0.15.15)

[warning] String contains ambiguous `．` (FULLWIDTH FULL STOP). Did you mean `.` (FULL STOP)?

(RUF001)